Medical Treatment - Add Additional Conditions for PAK Use (#8964)
acemod/ACE3: docs/wiki/feature/medical-system.md

Time Coefficient PAK: This setting controls how long a PAK takes to apply. Set to higher values to increase, lower to decrease. PAK has a minimum of 10 seconds usage time.

> Required Wounds State PAK: Controls whether a patient's wounds must be bandaged or stitched in order to use a PAK.

Reviewer: ```suggestion
Required Wounds State PAK: Controls whether a patient's wounds must be bandaged or stitched in order to use a Personal Aid Kit.
```

> Allow Pain PAK: Controls whether PAKs can be used on patients in pain.

Reviewer: ```suggestion
Allow Pain PAK: Controls whether Personal Aid Kits can be used on patients in pain.
```

> Min Blood PAK: Minimum blood volume percentage required to use a PAK.

Reviewer: ```suggestion
Min Blood PAK: Minimum blood volume percentage required to use a Personal Aid Kit.
```

> Min Heart Rate PAK: Minimum heart rate required to use a PAK.

Reviewer: ```suggestion
Min Heart Rate PAK: Minimum heart rate required to use a Personal Aid Kit.
```
